Week8/Day4/ExercisesXP/exercisesXP+.py

```diff
@@ -78,7 +78,7 @@ class TheIncredibles(Family):
     def use_power(self, member):
         if member['age'] >= 18:
             print(f"{member['name']} has {member['power']}")
-        else:
+        else: # this else is redundant
```

Comment:
didn't get this, if I throw it away, this part of code will always end with Exception Error, no?